client/src/components/leftPanel/Tag.scss

```diff
@@ -4,7 +4,7 @@
   border: 0;
   background-color: transparent;
   flex-wrap: wrap;
-  width: 100%;
+  width: 184px;
```

Comment:
This didn't help me, I still see the unnecessary horizontal scroll bar.

I see it only when the vertical scroll bar is also visible (i.e. when there is a lot of items).